Services/OperationService/Controllers/OperationsController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using DTO;
+using DTO.MarketBrokerObjects;
+using System.Threading.Tasks;
+using Interfaces;
+using DTO.BrokerRequests;
+
+namespace OperationService.Controllers
+{
+    [ApiController]
```

Comment:
Why do we use ApiControlller? Maybe we can use Controller?

Reply:
We use Api controllers in all services